src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -82,7 +82,12 @@
 {{#chip_endpoints}}
 endpoint {{endpointId}} {
   {{#chip_endpoint_clusters}}
-  instantiate cluster {{asUpperCamelCase name}};
+  {{!}} {{~#if (isServer side)~}}
```

Comment:
Why the `{{!}}` bit?

Reply:
I updated the code, but generally it is for space management:

```
  {{#if true~}}
  test 
  {{~/if~}}
   data
```
ends up being `testdata` because all the space is eaten away.
```
  {{#if true~}}
  test 
  {{~/if~}}
  {{!}} data
```
ends up being `test data` because the  space is removed only until the first comment (which is empty here).

Reply:
Mainly working around that I have no restyler for the idl, so I want codegen to always generate nice code.